nilearn/glm/contrasts.py

```diff
@@ -235,9 +236,11 @@ def stat(self, baseline=0.0):
         self.stat_ = stat.ravel()
         return self.stat_
 
+
     def p_value(self, baseline=0.0):
-        """Return a parametric estimate of the p-value associated
-        with the null hypothesis: (H0) 'contrast equals baseline'
+        """Return a parametric estimate of the p-value associated with
+        the null hypothesis (H0) where 'contrast equals baseline',
```

Comment:
what does the "where" mean here?

Reply:
It is a way to phrase what the null hypothesis means. But perhaps, it is not clear. What do you suggest?

Reply:
For me the previous phrasing
  "the null hypothesis (H0): 'contrast equals baseline'"
was clear, but if you prefer the new phrasing that's ok too!

Reply:
OK, I just changed it.